scripts/install_chrome_on_travis.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script should only be ran by Travis to install and provide a constant 
+# version of Chrome for Protractor.
+
+export CHROME_SOURCE_URL=https://raw.githubusercontent.com/webnicer/chrome-downloads/master/x64.deb/google-chrome-stable_67.0.3396.99-1_amd64.deb
```

Comment:
Please don't hard code this -- that defeats the purpose of the environment variable. At the moment, we can simply change the Chrome version from the repository settings.
Is there a way to use the environment variable from Travis directly?

Reply:
We are using hardcoded version of dependencies in manifest.json and setup.sh

In this scenario, we are trying to install only this version of Chrome. I see no difference between setting it here and setting on Travis's settings.

Beside, this is the natural place developers going to look should Protractor ( or other dependency) decides to use a different version of Chrome and we need to downgrade our Chrome version again.

@seanlip , what do you think?

Reply:
> We do not know when we might need to do that and at what frequency -- at some point, this goes stale or the link breaks, we just don't need to send in a PR for this. I have been updating the Chromedriver for quite a few times now -- each time, just the version has to be incremented. 
With all due respect, I think the ease of not submitting PR does not justify using Travis environment variable here. 
There are definitely exemplary use cases of Travis's run-time environment variable, like the browserstack credentials, like API keys..., etc but I don't think Chrome version should be a Travis enviroment variable.

Reply:
Sorry, but I see maintenance and the ease of keeping the building process up as an important point. If the majority is otherwise, I am happy to go with the flow :)
Thanks!

Reply:
Here's my take: on the whole, since @apb7 is responsible for keeping the overall dev workflow process running smoothly, I think he should make the final decision here. But I also think that whatever decision is made should at least account for and address the concerns brought up from the other standpoint -- that helps to sanity-test the decision and ensure that it is the best one we can take, given the information that is available.

So: @apb7, what do you see as @hoangviet1993's concerns, and how would you address them? And, @hoangviet1993 -- same question for @apb7's concerns.

(Also, don't just identify the other position's concerns and say "I think X is more important" or "I just don't think we should do Y". Instead, read their comments carefully and try to make sure you understand the root cause of the other position's concern, then see if you can come up with a way to address that. This process might lead us to a third, better, solution, or a better understanding of the principles underlying one of the existing solutions.)

Once that's been fleshed out a bit more, if you still need me to weigh in, I'd be happy to do so.

Reply:
I originally made the install script to be usable by local environment. 

In retrospect, since I have a header clearly stating that the script is not supposed to be ran locally, my original intention just doesn't make sense anymore.

Sorry for all of the inconvenience cause. I have made changes complying with Travis's env var. 

PTAL @apb7

Reply:
Could someone write a description of the chat and the outcomes here? I feel like a reader of this thread is missing half the story :)

Reply:
Actually, just to clarify, we don't need the whole story. But @apb7 @hoangviet1993 let's please record the following for posterity to close out this thread properly:

- what decision was made
- why we took that decision
- (if applicable) how the other position's concerns were addressed

Thanks!

Reply:
Well I could start the thread by first taking all the blames.

I was pretty gung-ho about not having an env var for non-encrypted stuff.  I pretty much shut my brain down after that 2nd post.

After Sean's call to consider @apb7 situation more carefully, I came to a realization that was just me and my opinion and totally not cool for a team environment. 

I thus backed down from my stance. We didn't really have a spicy break-out verbal fight over Gitter or anything. Sorry to disappoint everyone looking for juicy juicy drama hehe 😄

Reply:
Oh no, sorry, that wasn't what I meant to imply at all! It's good to have these discussions since they give us more understanding of why we're doing what we're doing, and I was hoping to understand better the principles behind the opinion.

Also, to be clear, the summary wasn't meant for me -- I misspoke. It's for other folks who look at this travis script in the future, and wonder why we're doing things the way we are. We can then point them back to the comment thread.

But, from what you said, I guess we took the decision to leave it as an env var, and that the rationales are the ones @apb7 gave. That should suffice for posterity. Thanks!